ci/jenkins/test-vmc.sh

```diff
@@ -465,6 +465,35 @@ function deliver_antrea {
     done
 }
 
+function run_integration {
```

Comment:
We remove it from this script by purpose, I suggest you to add integration related to the new file but rename it from `test-unit.sh` to `test-unit-integration.sh`.